README.md

```diff
@@ -40,15 +49,20 @@ Train and test either a Naive Bayes Classifier (text_classifier), a Decision Tre
 
 ### Examples
 
+After installing medinify, run:
+
 ```bash
-# Naive Bayes Example
-python text_classifier.py -i citalopram_train.csv -s stopwords.txt -c neutral.txt -z 10 -cl nb
+from medinify.sentiment.review_classifier import ReviewClassifier
 
-# Decision Tree Example
-python text_classifier.py -i citalopram_train.csv -s stopwords.txt -c neutral.txt -z 10 -cl dt
+def main():
```

Comment:
We use main() to keep it more organized in our example file but you don't actually need it. This example can probably just have the three lines that init and run ReviewClassifier()